plat/rockchip/rk3368/aarch64/platform_common.c

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
This file and the corresponding file in rk3399 is same except for plat_mmap[] array. You could consider sharing this file and just separating the array between the 2 variants.